htdocs/core/ajax/flowjs-server.php

```diff
@@ -73,6 +74,9 @@
 
 if (!empty($conf->$module->dir_temp)) {
 	$upload_dir = $conf->$module->dir_temp;
+	if (!empty($uploaddirname)) {
+		$upload_dir .= "/".$uploaddirname;
```

Comment:
Must sanitize the uploaddirname with dol_sanitizeFileName().
Can be done line 61 just after getting paramvalue.